cmd/res/configuration.yaml

```
@@ -0,0 +1,96 @@
+Writable:
+  LogLevel: INFO
+  InsecureSecrets:
+    credentials001:
+      SecretName: credentials001
+      SecretData:
+        username: ''
```

Same change for others
```suggestion
        username: ""
```